controllers/decks/DeckController.go

```diff
@@ -30,6 +30,32 @@ func DeleteDeck(c *gin.Context) {
 	c.JSON(http.StatusOK, "deck deleted")
 }
 
+// DeleteCardFromDeck gets the specified deck data back
+// looks for the card to be deleted from the deck and removes it
+func DeleteCardFromDeck(c *gin.Context) {
+	deckID := c.Param("deckID")
+	cardID, _ := strconv.ParseInt(c.Param("cardID"), 10, 64)
```

Comment:
The ID field in the card struct is type primitive.ObjectID not integer, so this may not work. Additionally we will probably need to change the Deck struct to store Cards array of primitive.ObjectID.